syntax = "proto3";

// Definition der Nachrichten
message RobotStatus {
  bool is_active = 1;
  // Fügen Sie hier weitere Felder für den Zustand des Roboters hinzu
}

message RobotRegistration {
  string robot_id = 1;
  string robot_name = 2;  
}

// Definition des Dienstes
service RobotControl {
  rpc RegisterRobot(RobotRegistration) returns (RobotStatus);
  rpc GetRobotStatus(RobotRegistration) returns (RobotStatus);
  
}

